VMAccess/vmaccess.py

```diff
@@ -275,6 +276,16 @@ def _set_user_account_pub_key(protect_settings, hutil):
 
     # Reset ssh key with the new public key passed in or reuse old public key.
     if cert_txt:
+        # support for SSH2-compatible format for public keys in addition to OpenSSH-compatible format
+        if (cert_txt.strip().startswith(BeginSSHTag)):
+            ext_utils.set_file_contents("temp.pub", cert_txt.strip())
+            retcode, output = ext_utils.run_command_get_output(['ssh-keygen', '-i', '-f', 'temp.pub'])
```

Comment:
Is ssh-keygen available on all distros, or do we need to check for existence of it first?